src/opera/util/run_utils.py

```diff
@@ -131,9 +135,9 @@ def create_sas_command_line(sas_program_path, sas_runconfig_path,
             if not os.access(executable_path, mode=os.X_OK):
                 raise OSError(f"Requested SAS program path {sas_program_path} exists, "
                               f"but does not have execute permissions.")
-        # Otherwise, sas_program_path might be a python module path
         else:
-            command_line = ['python3', '-m', sas_program_path]
+            error_msg = "Configured executable from the RunConfig could not be found"
```

Comment:
Similar to the error message above, lets include `{sas_program_path}` somewhere in this message to assist with troubleshooting.

Same applies for the error message in `create_qa_command_line`